browser/css/jsdialogs.css

```diff
@@ -1967,6 +1971,10 @@ input[type='number']:hover::-webkit-outer-spin-button {
 #FormulaDialog .ui-tabs-content.jsdialog {
 	grid-auto-columns: 250px;
 	margin-right: 0.2em;
+	margin-top: 0px;
+}
+#FormulaDialog .ui-tab[id^='tabcontrol'] {
+	margin-bottom: 0px;
```

Comment:
Let's not add this. I would prefer to don't add special margin for special cases around tab control, at least for now

Reply:
DONE.